src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -197,18 +198,17 @@ describe('Exam assessment', () => {
                 cy.wait(examEnd.diff(dayjs(), 'ms'));
             }
             cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
-            const score = '5 of 10 points';
             // Sometimes the feedback fails to load properly on the first load...
-            cy.reloadUntilFound(`jhi-result:contains(${score})`);
-            cy.get('jhi-result').contains(score).should('be.visible');
+            cy.reloadUntilFound(`#result-score`);
+            cy.get('#result-score').should('contain.text', '5 of 10 points');
         });
     });
 
     function startAssessing() {
         artemis.pageobjects.assessment.course.clickExerciseDashboardButton();
         exerciseAssessment.clickHaveReadInstructionsButton();
         exerciseAssessment.clickStartNewAssessment();
-        cy.contains('You have the lock for this assessment').should('be.visible');
+        cy.get('#assessmentLockedCurrentUser').should('contain.text', 'You have the lock for this assessment');
```

Comment:
I think here we shouldn't assert the text in the element, but instead just make sure that the element is shown.
```suggestion
        cy.get('#assessmentLockedCurrentUser').should('be.visible');
```